Source/Core/DolphinQt/Config/NewPatchDialog.cpp

```diff
@@ -115,95 +140,91 @@ QGroupBox* NewPatchDialog::CreateEntry(PatchEngine::PatchEntry& entry)
   type_layout->addWidget(dword);
   type->setLayout(type_layout);
 
-  auto* offset = new QLineEdit;
+  auto* address = new QLineEdit;
   auto* value = new QLineEdit;
+  auto* comparand = new QLineEdit;
+
+  m_entries.emplace_back(std::make_unique<NewPatchEntry>());
```

Comment:
emplace_back returns a reference to the item so you don't have to get the last element manually:

```cpp
auto* new_entry = m_entries.emplace_back(std::make_unique<NewPatchEntry>()).get();
```